decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -101,14 +101,14 @@
       global = nil
     end
 
-    20.times do |n|
+    5.times do |n|
       author = Decidim::User.where(organization: feature.organization).all.sample
       user_group = [true, false].sample ? author.user_groups.verified.sample : nil
-      state, answer = if n > 15
+      state, answer = if n > 4
                         ["accepted", Decidim::Faker::Localized.sentence(10)]
```

Comment:
Looks like `n` is never be more than `4` if we do stuff 5 times :D. Thanks Codecov!